Lib/test/test_tkinter/test_misc.py

```diff
@@ -66,9 +71,9 @@ def test_tk_busy(self):
         f.tk_busy_forget()
         self.assertFalse(f.tk_busy_status())
         self.assertFalse(f.tk_busy_current())
-        with self.assertRaisesRegex(TclError, "can't find busy window"):
+        with self.assertRaisesRegex(TclError, messages['no_busy']):
             f.tk_busy_configure()
-        with self.assertRaisesRegex(TclError, "can't find busy window"):
+        with self.assertRaisesRegex(TclError, messages['no_busy']):
```

Comment:
Since we're already using `assertRaisesRegex` here, I think we'd be better off just adjusting the regex rather than maintaining two separate sets of messages:
```suggestion
        with self.assertRaisesRegex(TclError, r"can('| ?no)t find busy window"):
```

Similar elsewhere.

Reply:
Leaving the PCbuild changes for another PR is fine with me.  I needed to test on Windows.  So I needed to build
_tkinter on Windows.  I was trying to leave some breadcrumbs so others would know what is involved in building against Tk 9 on Windows.  I guess I didn't choose the best way to do that.

Reply:
Those are valuable breadcrumbs that we don't want to lose :).  I do want us to update the Windows build to 9.0 as soon as we reasonably can, but we'll need to decide whether we're jumping there forever or maintaining support for 8.6 in `PCbuild/` that we ourselves won't be using.  (Personally, I think I'm fine with the direct jump in 3.14, but this is mostly going to be up to @zooba's decision.)

Reply:
Here is the issue with the regexes, as far as I am concerned.  Unfortunately, there have been many minor changes to the Tk error messages made recently, along the lines of can't -> can not.  That seems to be ongoing.  Maybe it will be easy to write conditional regexes which accept all of the variants.  I wasn't sure.  So I thought that having a catalog might be simpler than relying on regex cleverness.  I agree that, so far, regexes are easily able to accommodate the minor changes to error messages.  I don't know how long the error message tinkering will persist for.

A related topic, which I ignored for this PR, is that the Tcl/Tk project made a big effort to remove the "master-slave" terminology from Tcl and Tk, with considerable internal controversy.  However, tkinter is still using master and slave.  It could be using parent and child instead and be more consistent with Tcl/Tk.

Reply:
My current preference is to keep supporting 8.6 for a while, I'm not sure how long for. But since 9.0 isn't officially released yet we certainly can't stop supporting 8.6 yet, and even after they do then we'll want to make sure it's stable and compatible enough for all people who may be building from source (which includes myself, and while we haven't thoroughly audited the 8.6 sources, arguably we should have - being forced into upgrading to 9.0 and re-auditing the sources is a cost that we'd definitely want to defer).

Reply:
Changes to PCbuild have been reverted.  I renamed the modified versions and added them to the repo, as examples.